python/tk_desktop/licenses.py

```diff
@@ -27,5 +27,5 @@ def __init__(self, parent=None):
         self.ui = licenses.Ui_Licenses()
         self.ui.setupUi(self)
         # setSource seems broken on Qt4, so use setHtml instead.
-        with open(LICENSE_LOCATION) as f:
+        with open(LICENSE_LOCATION, encoding="utf-8", errors="replace") as f:
```

Comment:
I don't think we should add the error parameter because we distribute this file in tk-desktop so we don't expect encoding errors

```suggestion
        with open(LICENSE_LOCATION, encoding="utf-8") as f:
```